src/test/regress/expected/multi_simple_queries.out

```diff
@@ -664,6 +664,80 @@ DEBUG:  query has a single distribution column value: 1
  41 |         1 | aznavour     |      11814
 (5 rows)
 
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (0);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM articles TABLESAMPLE BERNOULLI (0);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (90) REPEATABLE (200);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+    50
```

Comment:
Since everything fits on a single page, this is giving a single page from each shard.

Reply:
added a comment